src/Microsoft.IdentityModel.Tokens/TokenHandler.cs

```diff
@@ -53,8 +54,23 @@ public int TokenLifetimeInMinutes
             set => _defaultTokenLifetimeInMinutes = (value < 1) ? throw LogExceptionMessage(new ArgumentOutOfRangeException(nameof(value), FormatInvariant(LogMessages.IDX10104, LogHelper.MarkAsNonPII(value)))) : value;
         }
 
+        /// <summary>
+        /// Gets the <see cref="JsonTokenType"/> of the token.
+        /// </summary>
+        public JsonTokenType TokenType { get; }
```

Comment:
I don't follow this one.

Reply:
A TokenHandler is the base class for Saml, Saml2 and JWT tokens.
To call out JsonTokenType, doesn't fit here.